### ISSUE TITLE
cannot get tiger_cat results in the paper

### ISSUE THREAD

**Issue by yuxu915** [dvirsamuel/SeedSelect#5]
Hi, it's really good work! I try follow the readme in this repo and try to generate iamge of "tiger_cat", but I cannot get the satisfied results as shown in the paper. Specifically, loss plateaus at iter=5, and get resluts like 
![tiger_cat_optimized_res_5](https://github.com/dvirsamuel/SeedSelect/assets/94051337/cd190d09-69c6-43fc-af15-1956fb12bd5a)
notice that I save `image_pil` as image in `main.py` in line#92

**Comment by dvirsamuel** [comment]
Hi, we have fixes a small issue with the code.
Please try again.
Dvir

**Comment by yueanga** [comment]
still can not generate satisfied results, and my results are as former as well.

**Comment by danielamassiceti** [comment]
HI @dvirsamuel. Just adding here that I am also unable to repro the results in the paper. Can you advise if there is a bug or if the wrong hyperparameters are in the repo (e.g. learning rate?).
